{"identifier":{"url":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/unknown","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"]]},"variants":[{"paths":["\/documentation\/mdots\/autherror\/unknown"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"}],"roleHeading":"Case","role":"symbol","symbolKind":"case","modules":[{"name":"MDots"}],"title":"AuthError.unknown","externalID":"s:5MDots9AuthErrorO7unknownyA2CmF"},"kind":"symbol","primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"references":{"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError":{"navigatorTitle":[{"text":"AuthError","kind":"identifier"}],"title":"AuthError","url":"\/documentation\/mdots\/autherror","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"AuthError","kind":"identifier"}],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError","role":"symbol","type":"topic","abstract":[{"type":"text","text":"An enumeration representing various authentication errors that can occur."}]},"doc://com.estelapasjak.Mdots/documentation/MDots":{"kind":"symbol","abstract":[],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","url":"\/documentation\/mdots","type":"topic","images":[{"identifier":"new-icon.png","type":"icon"}],"title":"MDots","role":"collection"},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError/unknown":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}],"kind":"symbol","type":"topic","url":"\/documentation\/mdots\/autherror\/unknown","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/unknown","abstract":[],"title":"AuthError.unknown","role":"symbol"},"new-icon.png":{"identifier":"new-icon.png","alt":"The logo representing the MDot sensor application.","type":"image","variants":[{"url":"\/images\/new-icon.png","traits":["1x","light"]}]}}}